### PR TITLE
User modeling

### PR DESCRIPTION
**Pull request by ninhnv-1216**
## Related Tickets
- [#45568](https://edu-redmine.sun-asterisk.vn/issues/45568)

## WHAT (optional)
- User profile interface and profile update

## WHY (optional)
Screenshot from 2022-03-01 10-52-10
Screenshot from 2022-03-01 10-51-16

### CODE REVIEW

**Comment by ninhnv-1216** [comment]
ready

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,30 +1,83 @@
...
-          <th scope="col"><%= t "address" %></th>
-          <td scope="col">        
-            <label><%= @user.address %></label>
-          </td>
-        </tr>
-    </table>
-  </h2>
+<div class="container">
+  <div class="row">
+      <div class="col-12">
```
Comment (ghost) [inline app/views/users/show.html.erb]: indent sai em ơi

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ninhnv-1216** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**